### PR TITLE
run linter locally

### PR DESCRIPTION
**Pull request by zreigz**
## Summary
There are two ways to run linter locally:
`$ make lint`

## Checklist

- [ ] If required, I have updated the Plural documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] I have added a meaningful title and summary to convey the impact of this PR to a user.
- [ ] I have added relevant labels to this PR to help with categorization for release notes.

### CODE REVIEW

File: hack/run-linter.sh [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+set -euo pipefail
+
+cd $(dirname $0)/..
+
+source hack/lib.sh
+
+CONTAINERIZE_IMAGE=golang:1.18.4 containerize  ./hack/run-linter.sh
+
+go run github.com/golangci/golangci-lint/cmd/golangci-lint@latest run
```
Comment (maciaszczykm) [inline hack/run-linter.sh:11]: Use v1.46.2 like here: https://github.com/pluralsh/plural-cli/blob/master/.github/workflows/ci.yaml#L54

File: Makefile [hunk 2]
```diff
@@ -72,4 +72,7 @@ test: .PHONY
 	go test -v -race ./pkg/... ./cmd/...
 
 format: .PHONY # formats all go code to prep for linting
-	gofmt -s -w .
\ No newline at end of file
+	gofmt -s -w .
+
+linter: .PHONY
```
Comment (maciaszczykm) [inline Makefile:77]: Can you name it `lint`?

**Review by maciaszczykm — CHANGES_REQUESTED** [review]

File: hack/run-linter.sh (hunk 1 above)
Comment (maciaszczykm) [reply hack/run-linter.sh:11]: There is official Docker image that you can use: https://golangci-lint.run/usage/install/#local-installation

File: Makefile (hunk 2 above)
Comment (zreigz) [reply Makefile]: done

**Review by maciaszczykm — APPROVED** [review]

**Pull request merged**